lib/moment-range.js

```diff
@@ -329,24 +329,19 @@ DateRange.prototype.diff = function(unit) {
  * @return [moment, moment]
  */
 DateRange.prototype.parseRange = function (start, end) {
-  var parts;
-  var s = start;
-  var e = end;
+if(start === null) start = undefined;
+if(end === null) end = undefined;
 
-  if (arguments.length === 1 || end === undefined) {
+    if (arguments.length === 1 || end === undefined) {
```

Comment:
try 
```
DateRange.prototype.parseRange = function (start, end) {
 if (arguments.length === 1) {
```
and then just remove the unit tests that pass in null, we dont need to support that